controller/backing_image_manager_controller.go

```diff
@@ -1151,9 +1138,7 @@ func (c *BackingImageManagerController) isMonitoring(bimName string) bool {
 }
 
 func (m *BackingImageManagerMonitor) Run() {
-	m.log.Info("Starting monitoring")
```

Comment:
There is no need to remove the log or change the log level for bids monitor starting/stopping since you are still using info log for the bim monitor

Reply:
This change here is to move this info log outside of goroutine 
it is moved to here: https://github.com/longhorn/longhorn-manager/pull/1923/files#diff-50cf5d5b10c83f46186aef3f483f4c18054476a8370822b5f8f3ac692f64b66fR1104